tools/setup_permissions.sh

```diff
@@ -56,6 +65,12 @@ function grant_task_execution() {
          "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:master:task:user:root" \
          -d '{"description":"Execute Mesos tasks as user=root"}' \
          -H 'Content-Type: application/json'
+    # XXX see above
+    curl -k -L -X PUT \
+         -H "Authorization: token=${ACS_TOKEN}" \
+         "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:agent:task:user:root" \
```

Comment:
Yes, previous lines 49-58 were wrong. wrt mesos task ace's, the frameworks need the following:

| ace | purpose |
|-----|---------|
| dcos:mesos:master:task:user:${LINUX_USER} | may launch the scheduler on a master |
| dcos:mesos:agent:task:user:${LINUX_USER} | may launch the scheduler or pod member on an agent |

For each of ^^, there also needs to be a ".../users/${FRAMEWORK_NAME}-principal" entry so the (dc/os concept) service account can perform the action.